packages/blade/src/components/Amount/_decisions/decisions.md

```diff
@@ -0,0 +1,67 @@
+# Amount <!-- omit in toc -->
+
+Amount component is a UI element that displays and formats various currency values in a specified format.
+
+<img  src="./amount-thumbnail.png"  width="380" />
+
+## Design
+
+- [Figma - Amount](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=28012%3A580578&t=3peAz8A2n2Gw4WMl-1)
+
+## API
+
+| Prop           | Type                                                               | Default     | Description                                                                    | Required |
+| -------------- | ------------------------------------------------------------------ | ----------- | ------------------------------------------------------------------------------ | -------- |
+| children       | `string`                                                           | `undefined` | The value to be rendered within the component                                  | ✅       |
+| intent         | `default`, `positive`, `negative`, `notice`, `information`, `blue` | `default`   | The variant of the amount to be rendered                                       |          |
+| contrast       | `low`, `high`                                                      | `low`       | The contrast of the amount to be rendered                                      |          |
+| fontWeight     | `bold`, `regular`                                                  | `bold`      | The weight of the amount to be rendered                                        |          |
+| hasDecimals    | `true`, `false`                                                    | `true`      | Indicates whether the amount has or can have decimals                          |          |
+| hasTextPostfix | `true`, `false`                                                    | `true`      | Indicates whether a text postfix should be used                                |          |
```

Comment:
Can there be both text postfix and decimals? If not I think both these props should be clubbed in a way that doesn't allow impossible combinations.

Reply:
It is not impossible to have both postfix and decimal, we can have 21.20 l as a value.

Reply:
Is this case documented on Figma 🤔

Reply:
I couldn't find this particular case on figma, where it shows both decimals and a suffix like `100.00 k`

Reply:
Wondering if we can consolidate decimals and suffix usecases in this case.

Reply:
@divyanshu013 I think we are settling at using `hasSuffix` as the only prop for both decimals and humanize. Looking at design we will not be supporting both decimal and suffix at the same time.

Reply:
Added a comment [below](https://github.com/razorpay/blade/pull/993#discussion_r1108096596)